tensorflow/python/ops/math_ops.py

```diff
@@ -3212,6 +3218,15 @@ def sigmoid(x, name=None):
 
   Returns:
     A Tensor with the same type as `x`.
+    
+  Usage Example:
+
+  >>> y = tf.math.sigmoid(10.0)
```

Comment:
Use smaller numbers like 1 and -1 that let you use fewer digits of precision (so replace 0.99999546 with 0.9... etc)

Reply:
Would you like me to make both changes you requested (change inputs, use fewer digits of precision on outputs) or only one? I was trying to demonstrate how the values approach 0 and 1.